superset-frontend/packages/superset-ui-chart-controls/src/components/ControlSubSectionHeader.tsx

```diff
@@ -20,7 +20,7 @@ import { styled, css } from '@superset-ui/core';
 
 export const ControlSubSectionHeader = styled.div`
   ${({ theme }) => css`
-    font-weight: ${theme.fontWeightStrong};
+    font-weight: ${theme.fontWeightMedium};
```

Comment:
isn't `fontWeightMedium` the default? Can we simply remove the styling or are we cascading a stronger font-weight?

Less styling is better!